CHANGELOG.md

```diff
@@ -77,6 +77,7 @@
 - Added support for `DataFrame.corr`.
 - Added support for `DataFrame.equals` and `Series.equals`.
 - Added support for `DataFrame.reindex` and `Series.reindex`.
+- Added support for `Index.equals`.
```

Comment:
I think this needs to be moved to the new release above.

Reply:
good catch! moved.